tools/tool-testing/selftest.js

```diff
@@ -1605,11 +1611,19 @@ var getFilteredTests = function (options) {
   if (! files.inCheckout()) {
     tagsToSkip.push('checkout');
   }
-  if (options.offline) {
-    tagsToSkip.push('net');
-  }
-  if (! options.includeSlowTests) {
-    tagsToSkip.push('slow');
+  if (options.galaxyOnly) {
+    tagsToSkip.push('non-galaxy');
+  } else {
+    tagsToSkip.push('galaxy');
+    if (options.offline) {
+      tagsToSkip.push('net');
+    }
+    if (! options.includeSlowTests) {
+      tagsToSkip.push('slow');
+    }
+    if (! options.galaxyOnly) {
```

Comment:
Heh, oops, will do. (I haven't addressed any of your comments or looked at them particularly since you said that you weren't done)